src/api/overlays/components/OverlayComponent.vue

```diff
@@ -174,6 +175,9 @@
         inject: ['dismiss', 'element', 'buttons', 'dismissable'],
         mounted() {
             this.$refs.element.appendChild(this.element);
+            setTimeout(() => {
```

Comment:
So is this necessary because we're waiting for the content of the overlay to render before we can focus it?

Reply:
@akhenry 
Yeah, 
Now as I am learning more about vue, wondering `this.$nextTick(` makes more sense instead of setTimeout?

Reply:
yes and now using `this.$nextTick` instead of `setTimeout`